db/structure.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.13  Distrib 5.6.35, for osx10.11 (x86_64)
+-- MySQL dump 10.13  Distrib 5.7.22, for osx10.13 (x86_64)
 --
--- Host: localhost    Database: usasearch_development
+-- Host: 127.0.0.1    Database: usasearch_test
```

Comment:
I have no idea what's going on here, but this `structure.sql` contains some bad info...

- We don't dump the `usasearch_test` DB
- We aren't using MySQL 5.7.x

Reply:
@ethanewing , let's chat about this in our meeting.

Reply:
@ethanewing , please install [MySQL 5.6](https://dev.mysql.com/doc/refman/5.6/en/installing.html)

Reply:
@ethanewing - don't worry about switching to 5.6 for now. That can happen after we wrap up this story.